vendor/bat-native-ads/src/bat/ads/internal/ml/transformation/hashed_ngrams_transformation.h

```diff
@@ -0,0 +1,44 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_ML_TRANSFORMATION_HASHED_NGRAMS_TRANSFORMATION_H_
+#define BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_ML_TRANSFORMATION_HASHED_NGRAMS_TRANSFORMATION_H_
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "bat/ads/internal/ml/transformation/transformation.h"
+
+namespace ads {
+namespace ml {
+namespace transformation {
+
+class HashVectorizer;
+
+class HashedNGrams : public Transformation {
```

Comment:
`HashedNGrams` should be `HashedNGramsTransformation`, same for other namespaces which have same name for object

Reply:
Done